lib/dal/tests/integration_test/old-engine-testing-layout/internal/attribute/prototype.rs

```diff
@@ -515,8 +515,7 @@ async fn remove_component_specific(ctx: &DalContext) {
                 assert!(
                     AttributePrototype::get_by_id(ctx, confirm_deletion_prototype_id)
                         .await
-                        .expect("could not get attribute prototype by id")
-                        .is_none()
```

Comment:
This code isn't compiling right? Looks like a find and replace function caught it. We should probably remove this change, but it doesn't matter.

Reply:
Nope, we were returning Option in get_by_id, but we always returned Some. So I made AttributePrototype::get_by_id work like other get_by_ids in the new engine, returning an error if missing. And this was the only place looking for a non existant prototype. So instead of none is error.